katas/langtonant/daniel/dani_ant.test.js

```diff
@@ -0,0 +1,43 @@
+import { findNewdirection  } from './dani_ant';
+
+test ('This part should give new direction to the ant based on color and current position', ()=>{
+    let newDirection = findNewdirection(0, 'up');
+    expect (newDirection).toBe('right');
+
+    newDirection = findNewdirection(0, 'right');
+    expect (newDirection).toBe('down');
+
+    newDirection = findNewdirection(0, 'down');
+    expect (newDirection).toBe('left');
+
+    newDirection = findNewdirection(0, 'left');
+    expect (newDirection).toBe('up');
+
+    newDirection = findNewdirection(1, 'up');
+    expect (newDirection).toBe('left');
+
+    newDirection = findNewdirection(1, 'left');
+    expect (newDirection).toBe('down');
+
+    newDirection = findNewdirection(1, 'down');
+    expect (newDirection).toBe('right');
+
+    newDirection = findNewdirection(1, 'right');
+    expect (newDirection).toBe('up');
+});
+
+import { antMoveCorrectDir } from './dani_ant';
```

Comment:
Please set all imports at the top of the file, this will make them easier to find and also, it is a more common practice in most development teams